packages/chain-bridge/src/adapters/LayerZeroAdapter.sol

```diff
@@ -44,21 +44,15 @@ contract LayerZeroAdapter is OAppRead, IBridgeAdapter, BaseBridgeAdapter {
     /// @notice Inverse mapping of LayerZero chain IDs to our chain IDs
     mapping(uint32 lzEid => uint16 chainId) public lzEidToChain;
 
-    /// @notice Message type for state read
-    uint16 public constant STATE_READ = 2;
-
-    /// @notice Message type for general message
-    uint16 public constant GENERAL_MESSAGE = 3;
-
-    /// @notice Mapping of message types to their minimum gas limits
-    mapping(uint16 msgType => uint128 minGasLimit) public minGasLimits;
-
     /// @notice Read channel identifier for lzRead operations
     uint32 public constant READ_CHANNEL_THRESHOLD = 4294965694; // Used to identify responses
 
     /// @notice Active read channel ID for sending read requests
     uint32 public readChannelId;
 
+    /// @notice Minimum gas limit for operations
+    uint128 public minGasLimit;
```

Comment:
Have added gas limit to registry in another PR. So, we can just use a standard gas limit via config managed that applies across the board. Think simplicity here trumps have adapter specific overrides especially given excess is refunded anyway.

Reply:
ok, leaving it as is so it can be removed later